packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -810,16 +810,16 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
         this.quickOpenService.open({
             onType: (_, accept) => accept(items)
         }, {
-                placeholder: 'Select File Icon Theme',
```

Comment:
please revert this file, there is no changes except whitespaces

Reply:
File should be reverted now

Reply:
File should be reverted now